en/frames_rover/differential_rover.md

```diff
@@ -32,82 +32,247 @@ To start using the differential-steering rover:
 
 3. Open the [Actuators Configuration & Testing](../config/actuators.md) to map the motor functions to flight controller outputs.
 
-This is sufficient to drive the the rover in [manual mode](../flight_modes_rover/index.md#manual-mode) (see [Flight modes](../flight_modes_rover/index.md)).
+This is sufficient to drive the the rover in [manual mode](../flight_modes_rover/manual.md#manual-mode) (see [Flight modes](../flight_modes_rover/index.md)).
 
 ::: info
-The parameter [RD_MAN_YAW_SCALE](../advanced_config/parameter_reference.md#RD_MAN_YAW_SCALE) can be used to scale the manual input for the yaw rate.
+The parameter [RD_MAN_YAW_SCALE](#RD_MAN_YAW_SCALE) can be used to scale the manual input for the yaw rate.
 :::
 
-## Tuning (Basic)
+## Tuning Process
 
-This section goes through the basic parameters that need to be set to use all other features for the differential-steering rover.
-Navigate to [Parameters](../advanced_config/parameters.md) in QGroundControl and set the following parameters:
+To use the other supported drive modes for differential rovers, certain parameters have to be set and tuned.
+The following is intended as a step-by-step guide to set up your rover, each successive step will enable a drive mode with more autopilot support and features.
 
-1. [RD_WHEEL_TRACK](../advanced_config/parameter_reference.md#RD_WHEEL_TRACK) [m]: Measure the distance from the center of the right wheel to the center of the left wheel.
+::: warning
+For a mode to work properly **all** previous/lower modes must already have been configured!
+:::
+
+### Acro Mode
+
+To set up [Acro mode](../flight_modes_rover/manual.md#acro-mode) navigate to [Parameters](../advanced_config/parameters.md) in QGroundControl and set the following parameters:
```

Comment:
For all modes, at this point a reader doesn't know what acro mode is, so it wouldn't be obvious why/when you would do this. I would prefix each section with a one line summary about the mode. Something _like_:

```suggestion
### Acro Mode

[Acro mode](../flight_modes_rover/manual.md#acro-mode) is a minimally stabilised mode where the vehicle attempts to hold its rate of turn (but does not stablise heading or speed).

To set up this mode navigate to [Parameters](../advanced_config/parameters.md) in QGroundControl and set the following parameters:
```